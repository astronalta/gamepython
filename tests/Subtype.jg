
void() {
}

function() String {
    ret void()
}

