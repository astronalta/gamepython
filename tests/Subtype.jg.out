Subtype.jg:6:5: Return must conform to type 'String'
